ci-operator/step-registry/api-job/api-job-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+prow_api_token=$(cat "/var/run/vault/tests-private-account/prow-api-token")
+export APITOKEN=${prow_api_token}
+
+github_token=$(cat "/var/run/vault/tests-private-account/token-git")
+export GITHUB_TOKEN=${github_token}
```

Comment:
@jianzhangbjz add
```shell
export PATH=$PATH:/usr/local/bin
```
it will resolve error `/bin/bash: line 16: job: command not found`